packages/components/src/InputPhoneNumber/InputPhoneNumber.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { FormField, FormFieldProps } from "../FormField";
+
+export interface RegionSettings {
+  countryCode?: number;
+  format?: string;
+}
+
+export interface AllowedRegions {
+  NorthAmerica: RegionSettings;
+  GreatBritain: RegionSettings;
+  Unknown: RegionSettings;
+}
+
+const REGION_SETTINGS: AllowedRegions = {
+  NorthAmerica: { countryCode: 1, format: "(###) ###-####" },
+  GreatBritain: { countryCode: 44 },
```

Comment:
I think this relates to my comment on the main description. Is there a reason we didn't add format for Great Britain?

Reply:
@kkimble006 From the reading that I was doing, we can't guarantee the number of digits in a GB phone number, so the formatting could change dynamically as the number is added. 

As a result I think formatting a GB number would be very, very hard. Instead I chose to just prefix it with `+44 `. We could try and do dynamic typing but I think that's an iteration on it's own and would get complicated.

Reply:
Here are all the formats:

![GB formats](https://user-images.githubusercontent.com/779421/143947435-ed745476-1f56-4617-84c5-4a04641b47bb.png)

The varying number of digits inside the `(##)` and the digits in each gruoping afterward would make this super difficult.

Reply:
@andrewjdmccann should the "+44" still appear in the field? Like the "+1"?

Reply:
It should it's a bug if it isn't. Taking a look now.

Reply:
Let me know when that is working and I'll get more 👀 back on this PR!

Reply:
@kkimble006 Fixed. The +44 will now show up before typing a number.